sdk/src/feature_set.rs

```diff
@@ -380,7 +376,6 @@ lazy_static! {
         (do_support_realloc::id(), "support account data reallocation"),
         (prevent_calling_precompiles_as_programs::id(), "prevent calling precompiles as programs"),
         (optimize_epoch_boundary_updates::id(), "optimize epoch boundary updates"),
-        (remove_native_loader::id(), "remove support for the native loader"),
```

Comment:
Hey, can you please leave old features in the list so that the feature set hash only changes when new features are added? It's also nice to view a history of all the activated features sometimes with `solana feature status`

Reply:
Roger that, shall I add it back in?

Reply:
Yeah, that'd be great thanks!